uktrade/data-hub-frontend
# Add link to remove all filters

## Description of change
This feature allows the user to remove all filters by clicking on a single link.

## Test instructions
Go to the investment project collection list page, search for an adviser then click on the "remove all filters" and the adviser should be removed. All we are really looking at here is that the url is set back to the pathname which is `/investments/projects` 

## Screenshots
![Screenshot 2020-11-19 at 10 47 49](https://user-images.githubusercontent.com/10154302/99656700-2407a680-2a55-11eb-8a40-7a9dc6b0f690.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
